features/in_onderzoek.feature

```diff
@@ -615,26 +782,30 @@ Functionaliteit: in onderzoek
       | type                | RaadpleegMetBurgerservicenummer    |
       | burgerservicenummer | 555550001                          |
       | fields              | burgerservicenummer,verblijfplaats |
-      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats.inOnderzoek' gegevens
-      | naam                             | waarde     |
-      | aanduidingBijHuisnummer          | true       |
-      | datumIngangGeldigheid            | true       |
-      | datumVestigingInNederland        | true       |
-      | functieAdres                     | true       |
-      | huisletter                       | true       |
-      | huisnummer                       | true       |
-      | huisnummertoevoeging             | true       |
-      | nummeraanduidingIdentificatie    | true       |
-      | adresseerbaarObjectIdentificatie | true       |
-      | landVanwaarIngeschreven          | true       |
-      | straat                           | true       |
-      | postcode                         | true       |
-      | korteNaam                        | true       |
-      | woonplaats                       | true       |
-      | datumIngangOnderzoek.datum       | 2022-03-07 |
-      | datumIngangOnderzoek.type        | Datum      |
-      | datumIngangOnderzoek.langFormaat | 7 maart 2022 |
-
+      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde       |
+      | verblijfadres.huisnummer                                   | 70           |
+      | verblijfadres.straat                                       | spui         |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07   |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.type        | Datum        |
+      | verblijfadres.inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022 |
+      | verblijfadres.inOnderzoek.aanduidingBijHuisnummer          | true         |
+      | verblijfadres.inOnderzoek.huisletter                       | true         |
+      | verblijfadres.inOnderzoek.huisnummer                       | true         |
+      | verblijfadres.inOnderzoek.huisnummertoevoeging             | true         |
+      | verblijfadres.inOnderzoek.straat                           | true         |
```

Comment:
ook hier: aanduidingbijhuisnummer, huisletter en huisnummertoevoeging in onderzoek terwijl deze er niet zijn?

Reply:
hele categorie is in onderzoek, dus aan de juistheid van elk gegeven wordt getwijfeld. We hebben geen andere manier gespecificeerd om aan te geven dat de hele categorie in onderzoek staat.

Reply:
OK, prima